CHANGELOG.md

```diff
@@ -22,6 +22,14 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 - Config parameter `external_address` is now a required value. (#826)
 
+- `Connect`-messages have been removed from the protocol. Now handshake between
+  peers are handled by noise protocol. (#866)
+
+- New message type `PeersExchange` has been added. Peers sends it as the
+  response to `PeersRequest`. (#866)
+
+### New features
```

Comment:
I think this line should be removed. There's already a new feature section below and the following item is definitely a breaking change.